packages/spor-react/src/theme/utils/sr-utils.ts

```diff
@@ -0,0 +1,13 @@
+/** All the styles you need to hide something visually, while still making it available for screen readers */
+export const srOnly = {
```

Comment:
Et nyttig verktøy for å skjule ting visuelt, men fortsatt ha det tilgjengelig for skjermlesere.